boards/arm/nrf52840_pca10056/doc/nrf52840_pca10056.rst

```diff
@@ -46,21 +46,36 @@ hardware features:
 +-----------+------------+----------------------+
 | Interface | Controller | Driver/Component     |
 +===========+============+======================+
-| NVIC      | on-chip    | nested vectored      |
-|           |            | interrupt controller |
 +-----------+------------+----------------------+
-| RTC       | on-chip    | system clock         |
-+-----------+------------+----------------------+
-| UART      | on-chip    | serial port          |
+| ADC       | on-chip    | adc                  |
 +-----------+------------+----------------------+
 | GPIO      | on-chip    | gpio                 |
 +-----------+------------+----------------------+
+| I2C       | on-chip    | i2c                  |
++-----------+------------+----------------------+
 | FLASH     | on-chip    | flash                |
 +-----------+------------+----------------------+
-| RADIO     | on-chip    | Bluetooth            |
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| SPI       | on-chip    | spi                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
 +-----------+------------+----------------------+
 | RTT       | on-chip    | console              |
 +-----------+------------+----------------------+
+| TIMER     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
```

Comment:
Should we also have UARTE for nRF52-based peripherals?

Reply:
Good point. @anangl anything else that I might have overlooked similar to this?

Reply:
SPI/SPIM and TWI/TWIM